kombu/simple.py

```diff
@@ -134,13 +135,13 @@ def __init__(self, channel, name, no_ack=None, queue_opts=None,
         else:
             exchange = queue.exchange
             routing_key = queue.routing_key
-        consumer = messaging.Consumer(channel, queue)
+        consumer = messaging.Consumer(channel, queue, accept=accept)
         producer = messaging.Producer(channel, exchange,
                                       serializer=serializer,
                                       routing_key=routing_key,
                                       compression=compression)
         super().__init__(channel, producer,
-                         consumer, no_ack, **kwargs)
+                         consumer, no_ack, accept)
```

Comment:
Why has this lost the `**kwargs` passthrough? We still accept them in the method signature so I assume we still need to pass them along to be cooperative.